spec.md

```diff
@@ -280,10 +280,45 @@ import "google/protobuf/wrappers.proto";
 
 option go_package = "csi";
 
+extend google.protobuf.EnumOptions {
+  // Indicates that this enum is OPTIONAL and part of an experimental
+  // API that may be deprecated and eventually removed between minor
+  // releases.
+  bool alpha_enum = 1060;
+}
+extend google.protobuf.EnumValueOptions {
+  // Indicates that this enum value is OPTIONAL and part of an
+  // experimental API that may be deprecated and eventually removed
+  // between minor releases.
+  bool alpha_enum_value = 1060;
+}
 extend google.protobuf.FieldOptions {
   // Indicates that a field MAY contain information that is sensitive
   // and MUST be treated as such (e.g. not logged).
   bool csi_secret = 1059;
+
+  // Indicates that this field is OPTIONAL and part of an experimental
+  // API that may be deprecated and eventually removed between minor
+  // releases.
+  bool alpha_field = 1060;
+}
+extend google.protobuf.MessageOptions {
+  // Indicates that this message is OPTIONAL and part of an experimental
+  // API that may be deprecated and eventually removed between minor
+  // releases.
+  bool alpha_message = 1060;
+}
+extend google.protobuf.MethodOptions {
```

Comment:
Do we want to allow this?

My understanding after the call was that we allow new fields in the main spec on existing RPCs with a experimental tag. But we force new RPCs in to a separate new experimental service (potentially in a different file).